html/siptest.js

```diff
@@ -843,14 +853,14 @@ function doCall(ev) {
 		bootbox.alert('Please insert a valid SIP address (e.g., sip:pluto@example.com)');
 		$('#peer' + suffix).removeAttr('disabled');
 		$('#dovideo' + suffix).removeAttr('disabled');
-		$('#call' + suffix).removeAttr('disabled').click(function() { doCall(helperId); });
+		$('#call' + suffix).removeAttr('disabled').click(function(ev) { doCall(ev); });
```

Comment:
I think this was a bug, after a failed attempt to call a sip uri (due invalid sip uri), `doCall` was receiving the helperId instead of the event. Can you double check this?

Reply:
Yeah it's definitely a leftover from a previous version of that function, as `doCall` does expect an object and not an ID now. Your change seems fine, since `ev` here is whatever jQuery adds as the argument to `click` (so the button info IIUC), and not the `ev` that was passed to `doCall` in the first place.